vue/sbc-common-components/tests/unit/notification.services.spec.ts

```diff
@@ -0,0 +1,33 @@
+// users.test.js
+import Axios from 'axios'
+import NotificationServices from '../../src/services/notification.services'
+import { SessionStorageKeys } from '@/util/constants'
+
+jest.mock('axios', () => ({
+  get: jest.fn(),
+  all: jest.fn(),
+  spread: jest.fn()
+}), {
+  virtual: true
+})
+
+const API_URL = 'https://status-api-dev.pathfinder.gov.bc.ca/api/v1'
+
+describe('get status for notification service', () => {
+  const results: any = []
+  const mockAxiosSpreadResult = jest.fn()
+  beforeAll(() => {
+    // @ts-ignore
+    Axios.get.mockClear()
+    // @ts-ignore
+    Axios.all.mockResolvedValue(results)
+    // @ts-ignore
+    Axios.spread.mockReturnValue(mockAxiosSpreadResult)
+    sessionStorage.__STORE__[SessionStorageKeys.ApiConfigKey] = JSON.stringify({ VUE_APP_STATUS_ROOT_API: API_URL })
+    NotificationServices.getNotifications()
+  })
+
+  it('should call Axios.get once for status ', () => {
+    expect(Axios.get).toHaveBeenCalledWith(`${API_URL}/whatsnew`)
+  })
+})
```

Comment:
How about some tests that the component displays various notifications as expected? Including read and not-read, etc.

Reply:
This is actually a draft version and will have more function to add it in. I will move the api call to notify service and add the test cases later.

Reply:
ok